toshiba_ac/device/__init__.py

```diff
@@ -129,7 +130,7 @@ async def handle_cmd_fcu_from_ac(self, payload: dict[str, JSONSerializable]) ->
             await self.state_changed()
 
     async def handle_cmd_heartbeat(self, payload: dict[str, t.Any]) -> None:
-        hb_data = {k: int(v, base=16) for k, v in payload.items()}
+        hb_data = {k: struct.unpack("b", bytearray.fromhex(v))[0] for k, v in payload.items()}
```

Comment:
Please update to only use signed conversion for temperatures, other values seems to be unsigned. Also please include comment explaining it like in the suggestion below.
```suggestion
        # Use signed conversion for temperatures, unsigned otherwise.
        hb_data = {k: struct.unpack("b" if "Temp" in k else "B", bytes.fromhex(v))[0] for k, v in payload.items()}
```